python/surf/devices/ti/_Lmk04828.py

```diff
@@ -24,24 +21,14 @@
 import ast
 
 class Lmk04828(pr.Device):
-    def __init__( self,
-        name        = 'Lmk04828',
-        description = 'LMK04828 Module',
-        memBase     = None,
-        offset      = 0x00,
-        hidden      = False,
-        expand      = True,
-    ):
-        super().__init__(
-            name        = name,
-            description = description,
-            memBase     = memBase,
-            offset      = offset,
-            hidden      = hidden,
-            expand      = expand,
-        )
-        
-        self.sysrefMode = 2 # 2 pulse sysref mode, 3 continuouse sysref mode
+    def __init__(   self,
+            name        = 'Lmk04828',
```

Comment:
As long as we're updating this stuff, we should fix these.
The `name` parameter is unnecessary if it matches the class name. 
I would also argue that `'LMK04828 Module'` is somewhat useless as a description.
I'd be fine with getting rid of it so it defaults to `''`.

Reply:
@bengineerd: Feel free to do this change on this PR

Reply:
Done in 60fba3a